packages/main/src/themes/ValueStateMessage.css

```diff
@@ -0,0 +1,26 @@
+.ui5-input-valuestatemessage-root {
```

Comment:
Use parameters for color, font family, etc...

Reply:
Done.